src/styles/quiz/question.module.scss

```diff
@@ -11,11 +11,12 @@
     @include localize('font-size', toRem(100), toRem(80));
     @include localize('letter-spacing', 0, 0.18em);
   }
+
   p {
-    font-size: toRem(65);
-    line-height: 1.2;
-    margin-top: toRem(50);
-    padding: 0 toRem(100);
+    @include localize('font-size', toRem(60), toRem(45));
+    line-height: 1.5;
+    max-width: toRem(2500);
```

Comment:
@tnordberg Here's the change where I've set the max-width to be `2500px`. You may be right that it was happening because of cache. Let me know if the issue persists.